### PR TITLE
[ADD]Added a feature to group a sale and purchase journals' account move lines and sum debit/credit amounts in FEC Report

### PR DESCRIPTION
**Pull request by krunal267**
Related Task:
https://www.druidoo.io/web#id=1081&action=367&model=project.task&view_type=form&menu_id=237

pic1

pic2

pic3

### CODE REVIEW

File: louve_addons/fr_fec_extend/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- encoding: utf-8 -*-
+{
+    'name': 'France - FEC',
```
Comment (ivantodorovich) [inline louve_addons/fr_fec_extend/__openerp__.py]:
```suggestion
    'name': 'France - FEC Group Sale Purchase',
```

File: louve_addons/fr_fec_extend/wizard/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import fec
```
Comment (ivantodorovich) [inline louve_addons/fr_fec_extend/wizard/__init__.py]:
```suggestion
from . import fec

```

File: louve_addons/fr_fec_extend/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import wizard
```
Comment (ivantodorovich) [inline louve_addons/fr_fec_extend/__init__.py]:
```suggestion
from . import wizard

```

File: louve_addons/fr_fec_extend/wizard/fec_view.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        <field name="inherit_id" ref="l10n_fr_fec.account_fr_fec_view"/>
+        <field name="arch" type="xml">
+          <field name="date_to" position="after">
+            <field name="group_sale_purchase"/>
+          </field>
+        </field>
+    </record>
+
+  </data>
+</openerp>
```
Comment (ivantodorovich) [inline louve_addons/fr_fec_extend/wizard/fec_view.xml]:
```suggestion
</openerp>

```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Thanks @krunal267 :)

Minor suggestions (you can just click on apply changes on each)

Also, would you please rename the module to something more descriptive?
ie: `l10n_fr_fec_group_sale_purchase`

**Comment by ivantodorovich** [comment]
Wow, actually it turns out I'm able to do it from Github. Yay ^^
Only thing missing is the module rename

**Comment by krunal267** [comment]
Hi Ivan, 
Thanks for the suggestions :).
Module is renamed to l10n_fr_fec_group_sale_purchase.

**Comment by ivantodorovich** [comment]
:+1: thanks!! :)

awesome!

I'm closing this so I can retarget on https://github.com/AwesomeFoodCoops/odoo-production/pull/971

**Pull request closed without merging**